source/administration/changelog.md

```diff
@@ -4,6 +4,154 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.9
+
+Release date: 2018-04-16
+
+### Highlights
+
+#### Channel Mute
+ - Added a `/mute` command, meaning that when a channel is muted, desktop, push and email notifications are not sent for the channel.
+ 
+#### Teammate Name Display Setting
+ - Added rendering of at-mentions by the teammate name display.
+ 
+#### Team Icons
+ - Added support for team icons in the team sidebar, within the border of the existing team icons.
+
+#### Advanced Permissions ([Enterprise Edition E20](https://about.mattermost.com/pricing/))
+ - Migrated system permissions to roles, which allows for more granular permissions at team and channel level, as well as custom roles, in future Mattermost releases.
+ - Several configuration settings were migrated to roles in the database where changing their `config.json` values no longer take effect. See the compatibility section of this changelog to learn more.
+ 
+#### Global Relay (Beta) ([Enterprise Edition E20](https://about.mattermost.com/pricing/) Add-On)
+ - Added compliance export support for Global Relay as a compliance solution. [Learn more here](https://about.mattermost.com/default-compliance-export-documentation).
+
+### Improvements
+
+#### Web User Interface
+ - Users can now set their timezone in **Account Settings > Timezone**.
```

Comment:
"timezone" should be two words but I guess we should leave it here if "Timezone" is shown in "Account Settings"?